packages/now-cli/src/util/get-decrypted-env-records.ts

```diff
@@ -0,0 +1,50 @@
+import getEnvVariables from './env/get-env-records';
+import getDecryptedSecret from './env/get-decrypted-secret';
+import Client from './client';
+import { Output } from './output/create-output';
+import { ProjectEnvTarget, Project } from '../types';
+
+import { Env } from '@vercel/build-utils';
+
+export default async function getDecryptedEnvRecords(
+  output: Output,
+  client: Client,
+  project: null | Project,
```

Comment:
```suggestion
  project: Project,
```

Is there still a case when Project is null?